client/src/login/LoginForm.js

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+export const LoginForm = reduxForm({
```

Comment:
This looks like it's working again. I do think we should try and avoid exporting more than just the default when we can.